src/os/bluestore/BlueStore.cc

```diff
@@ -16121,7 +16121,7 @@ int BlueStore::_do_gc(
 int BlueStore::_do_write(
   TransContext *txc,
   CollectionRef& c,
-  OnodeRef o,
+  OnodeRef& o,
```

Comment:
I like optimization OnodeRef->OnodeRef&.
But would we not be better having it in different PR, for testing purposes?
Or is it somehow relevant to pin/unpin logic?

Of course plus side of having it here is that with one backport we will have perf improvement too.

Reply:
No IIRC it's not relevant to ref counting. And this apparently deserves a standalone commit/PR. But let's leave it as-is for now - I strongly want this fix to be merged ASAP - there is a plenty of user complains on the bug.